catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -214,6 +227,29 @@ protected Object executeWithSubject() throws Exception {
         return null;
     }
 
+    private StringBuilder getUserInputModifiable() throws IOException {
+        int in;
+        StringBuilder builder = new StringBuilder();
+        while ((in = session.getKeyboard()
+                .read()) != '\r') {
```

Comment:
Am I missing some great need to read character by character? Couldn't you just do something like...
```java
        try (BufferedReader reader = new BufferedReader(new InputStreamReader(session.getKeyboard()))) {
            String line = reader.readLine();
            getConsole().println(line);
            return new StringBuilder(line);
        }
```